### ISSUE TITLE
`WP_SENTRY_ERROR_TYPES` set to `E_ALL & ~E_NOTICE` still logs PHP user notices

### ISSUE THREAD

**Issue by vtwigt** [stayallive/wp-sentry#167]
Hi @stayallive ,

As mentioned in the title, I am using the following constant in `wp-config.php`:
```
define( 'WP_SENTRY_ERROR_TYPES', E_ALL & ~E_NOTICE );
```

Nevertheless, my Sentry account is being flooded by the following notices (over 1600 by now):
> User Notice: Function wp_cache_flush_runtime was called <strong>incorrectly</strong>. Your object cache implementation does not support flushing the in-memory runtime cache. Please see <a href="https://wordpress.org/documentation/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.1.0.)

I am not entirely sure from the [`README.md`](https://github.com/stayallive/wp-sentry/blob/v6.23.0/README.md#configuration), but does the Sentry constant use and support all PHP error constants, including `E_USER_NOTICE`?
In other words, should I have changed the configuration as follows?
```
define( 'WP_SENTRY_ERROR_TYPES', E_ALL & ~E_NOTICE & ~E_USER_NOTICE );
```

And if it does, may I suggest to add this information to the `README.md`?

**Comment by stayallive** [comment]
Well, technically the `E_USER_NOTICE` is different from `E_NOTICE` but if you also want to supress those than yes you should use `E_ALL & ~E_NOTICE & ~E_USER_NOTICE` you can see we have another example too:

> (Optionally) set the error types the PHP tracker will track:
> ```
> define( 'WP_SENTRY_ERROR_TYPES', E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_USER_DEPRECATED );
> ```

You can set any combination you like there just like you kan setup `error_types` for PHP itself. I didn't think it to be needed to be that specific saying it is using PHP constants since it is... but maybe it's better to be more explicit.

**Comment by vtwigt** [comment]
I agree that it works as intended, and I also agree that it can be interpreted from the example that the PHP error constants are being used, since WordPress is not defining these constants.

My point is that it would be super clear if an extra line in the `README.md` explains that all PHP error constants are supported. This way everybody knows that the ones mentioned in the example are not the only possibilities.

If you prefer to leave it as is, I understand. In both cases, feel free to close this issue. 👍

**Comment by stayallive** [comment]
Sure, updated it: 3cc874435766d3f3dd13e2dfe293b9ab6ff82843. Thanks!
